README.md

```diff
@@ -1 +1,120 @@
-# CodersCamp2021.Project.Fullstack
\ No newline at end of file
+
+# CodersCamp 2021 - Projekt Node.js
+
+<div align="center">
+
+[![Demo](https://img.shields.io/badge/-demo-green?logo=github)](https://coderscamp2021-hk-fullstack.herokuapp.com/)
+[![Figma](https://img.shields.io/badge/-brainstorming+wireframing-blueviolet?logo=figma)](https://www.figma.com/file/51gzaCTbilz6X9r0PZxmhX/Project.Fullstack?node-id=0%3A1)
+</div>
+
+<br/>
+
+<p>
+🍲🍝🍜 <strong>BierzCoChcesz</strong> 🍲🍝🍜 projekt skierowany do ludzi, którzy nie lubią gdy narzuca im się wybory jedzeniowe a jednocześnie zwracają uwagę na wartości odżywcze swoich posiłków.
+Aplikacja pozwala na skomponowanie przez użytkownika diety na cały dzień składającej się z dań z ulubionych restauracji. W praktyce to system partnerski zrzeszający najlepsze restauracje.</p>
+
+<br/>
+<br/>
+
+## 🖥️ Użyte technologie
+<br/>
+
+### Główne:
+ - [Docker](https://docs.docker.com/get-docker/)
+ - [MongoDB](https://www.mongodb.com/)
+ - [Mongoose](https://mongoosejs.com/)
+ - [NestJS](https://nestjs.com/)
+ - [Express](https://expressjs.com/)
+ - [Nodemailer](https://nodemailer.com/)
+ - [Swagger](https://swagger.io/tools/swaggerhub/?&utm_medium=ppcg&utm_source=aw&utm_term=swagger&utm_content=511173019632&utm_campaign=SEM_SwaggerHub_PR_EMEA_ENG_EXT_Prospecting&awsearchcpc=1&gclid=CjwKCAjwxOCRBhA8EiwA0X8hiwxh7NHt37o-DaApRMBtiZ25QonxHVTKTew0Pa2SYAC6nDVzJnYD3xoCejMQAvD_BwE&gclsrc=aw.ds)
+ - [Figma](https://www.figma.com/files/recent?fuid=1047244325467558592)
```

Comment:
Nie jestem przekonana czy ten link jest poprawny, jak ktoś jest niezalogowany to przenosi do logowania a podejrzewam że chciałaś dać link do głównej strony Figmy/link do naszej Figmy. Jeśli chodzi o link do naszej, to umieściłabym go w bardziej widocznym miejscu (i może dopisała co się tam znajduje), może w formie badge'a jak w poprzednich readme

Reply:
Linki:
FigJam: https://www.figma.com/file/51gzaCTbilz6X9r0PZxmhX/Project.Fullstack?node-id=0%3A1
Figma: https://www.figma.com/file/Cwxh4WHHLxBhhyxJQIgsI9/Project.Fullstack?node-id=10%3A13